File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/NeurophysiologicalAssessmentFormViewModel.cs
//$FB946FF7
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;

using Core.Controllers;

namespace Core.Controllers
{
    public partial class NeurophysiologicalAssessmentFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class NeurophysiologicalAssessmentFormViewModel: BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var neurophysiologicalAssessmentFormService = new NeurophysiologicalAssessmentFormServiceController();
            neurophysiologicalAssessmentFormService.NeurophysiologicalAssessmentFormInternal(this, objectContext);
            return (NeurophysiologicalAssessmentForm)objectContext.GetObject(this._NeurophysiologicalAssessmentForm.ObjectID, "NeurophysiologicalAssessmentForm");
        }
    }
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/NeurophysiologicalAssessmentFormViewModel.ts
//$EE21AC3B
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { NeurophysiologicalAssessmentForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class NeurophysiologicalAssessmentFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.NeurophysiologicalAssessmentFormViewModel, Core';
    @Type(() => NeurophysiologicalAssessmentForm)

    public _NeurophysiologicalAssessmentForm: NeurophysiologicalAssessmentForm = new NeurophysiologicalAssessmentForm();
}
